docs/calicoctl/bgp.md

```diff
@@ -0,0 +1,218 @@
+the peering associated with the specified peer IP address.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl bgp peer remove <PEER_IP>
+
+    <PEER_IP>:  The IP address (IPv4 or IPv6) of the BGP peer
+```
+
+Examples:
+
+```
+$ calicoctl bgp peer remove 192.0.2.10
+BGP peer removed from global configuration
+
+$ calicoctl bgp peer remove 2001:0db8::1
+BGP peer removed from global configuration
+```
+
+### calicoctl bgp peer show 
+This command displays the current list of configured global BGP peers.
+
+This command does not display the connection or protocol status of the peers.
+If you want to view that information, use the [`calicoctl status`](status.md)
+command.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl bgp peer show [--ipv4 | --ipv6]
+
+    --ipv4:  Optional flag to show IPv4 peers only
+    --ipv6:  Optional flag to show IPv6 peers only
+    
+    If neither --ipv4 nor --ipv6 are specified, all peers are displayed.    
+```
+
+Examples:
+
+```
+$ calicoctl bgp peer show
++----------------------+--------+
+| Global IPv4 BGP Peer | AS Num |
++----------------------+--------+
+| 192.0.2.10           | 64555  |
++----------------------+--------+
++----------------------+--------+
+| Global IPv6 BGP Peer | AS Num |
++----------------------+--------+
+| 2001:db8::1          | 64590  |
++----------------------+--------+
+ 
+$ calicoctl bgp peer show --ipv4
++----------------------+--------+
+| Global IPv4 BGP Peer | AS Num |
++----------------------+--------+
+| 192.0.2.10           | 64555  |
++----------------------+--------+ 
+```
+
+### calicoctl bgp node-mesh 
+This command is used to view the status of, or enable and disable, the full 
+node-to-node BGP mesh.
+
+When set to true, the Calico nodes automatically create a peering with all
+other Calico nodes in the deployment.
+
+In large deployments, you may want to set this value to be false in order to 
+manage BGP peerings explicitly.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl bgp node-mesh [on|off]
+
+    off:  Disable the node-to-node BGP mesh between all of the Calico nodes.
+    on:  Enable the node-to-node BGP mesh between all of the Calico nodes.
+    
+    If no parameter is specified, this command displays the current status.
+```
+
+Examples:
+
+```
+$ calicoctl bgp node-mesh on
+
+$ calicoctl bgp node-mesh off
+
+$ calicoctl bgp node-mesh
+off
+```
+
+### calicoctl bgp default
+This command is used to view and set the default AS number used by Calico 
+nodes.
+
+When a Calico node is started (see [`calicoctl node`](node.md) commands),
+the default AS number is used to configure the BGP peerings if one has not 
+been explicitly specified in the `calicoctl node` command.
+
+If any nodes are using the default AS number (i.e. it is not explicitly 
+specified), then changing the default value with the following command will
```

Comment:
"i.e. an AS number was not explicitly specified on that node"

Slightly clearer, since when I first read it I thought it was saying if the global default wasn't set.